sentinelsat/sentinel.py

```diff
@@ -910,6 +910,74 @@ def _download_online_retry(
 
         return ret_val
 
+    def download_quicklooks(self, products, directory_path=".", n_concurrent_dl=2):
+        """Download quicklook for a list of products.
+
+        Takes a dict of product IDs: product data as input. This means that the return value of
+        query() can be passed directly to this method.
+
+        File names on the server are used for the downloaded images, e.g.
+        "S2A_MSIL1C_20200924T104031_N0209_R008_T35WMV_20200926T135405.jpeg".
+
+        Parameters
+        ----------
+        products : dict
+            Dict of product IDs, product data
+        directory_path : string
+            Parent directory for quicklooks directory, where the downloaded images will be stored
+        n_concurrent_dl : integer
+            number of concurrent downloads
+
+        Returns
+        -------
+        dict[string, dict]
+            A dictionary containing the error of products where either
+            quicklook was not available or it had an unexpected content type
+        """
+
+        self.logger.info("Will download %d quicklooks", len(products))
+
+        outdir = join(directory_path, "quicklooks")
```

Comment:
Please remove the outdir (sub-directory) logic from the core function. Which ever directory the user passes to `download_quicklooks` should be the one they are written to. This might end up being the same directory the data files are downloaded to, but if a user needs to change that, they can use the Python API.